src/open_dread_rando/files/custom_scenario.lua

```diff
@@ -21,6 +21,18 @@ Scenario.tRandoHintPropIDs = {
     SHIP_1 = Blackboard.RegisterLUAProp("HINT_SHIP_1", "bool")
 }
 
+Scenario.tNumTanksMaxByScenario = {
+    s010_cave = 35,
+    s020_magma = 25,
+    s030_baselab = 23,
+    s040_aqua = 20,
+    s050_forest = 20,
+    s060_quarantine = 5,
+    s070_basesanc = 17,
+    s080_shipyard = 4,
+    s090_skybase = 0
```

Comment:
can you double check that we don't need to special case this one? it feels like it's just begging for a divide by zero error